docs/developer-docs/developer-welcome.md

```diff
@@ -0,0 +1,181 @@
+
+For details on how to propose a revision, see [Achievement Set Revisions](/guidelines/content/achievement-set-revisions)
+
+In the case where you are re-authoring the set (keeping the old achievements in unofficial and promoting your new reworked achievements, make sure to contact DevCompliance for approval) and you want to transfer over the earners for some/all of the previous achievements then you will need to reach out to RAdmin or #admin-site-cleanup on Discord to help you with this.
+
+The request would include the Game and a full list of achievement transfers in Old ID -> New ID format
+
+Example: 
+
+https://retroachievements.org/game/990
+
+Transfer earners for Game 990:
+
+11702 -> 175183
+
+11709 -> 175184
+
+And so on...
+
+## Emulator Updates
+
+You will now have a few new privileges in the RA emulators to take advantage of.
+
+### Creating Game Pages and Linking Hashes
+
+The biggest one is that now you can link new hashes to games by loading the game in an RA emulator and linking it.
+
+If creating a new page, select "New Title" and write an appropriate name in the text box. Check if there are any junk pages labeled ~Z~ which can be recycled. 
+
+::: info
+Further reading:
+
+[Creating Entries](/guidelines/developers/creating-entries)
+
+[Achievements for ROM Hacks](/guidelines/content/achievements-for-rom-hacks)
+
+[Subsets](/guidelines/content/subsets)
+
+[Working with the Right ROM](/guidelines/content/working-with-the-right-rom)
+:::
+
+![Creating a game page with an unidentified hash](/public/game-page-creation.png)
+
+### Code Notes
+
+You are now able to overwrite and delete any existing code notes. Be careful not to overwrite other’s notes without reason, if you are making improvements or correcting a note that is an acceptable reason.
+
+### Achievements
+
+Developers can directly make edits to achievements in Core.
+
+You can now freely promote and demote any achievements from Unofficial and Core through the emulator but it is much easier to do this through the site.
+
+## Site Updates
+
+You’ll now have a lot more access to tools on the site to help you as a developer.
+
+### Want to Develop Games
+
+Developers have the ability to add games to a Want to Develop list. Sets can be added through the Dev dropdown box on the game page.
+
+![Want to Develop Games button](/public/want-to-develop.png)
+
+### Game Page
+
+#### Creating Forum Topics
+
+When you go to claim a set, you no longer need to create a forum topic separately. The Make Claim button will automatically create a forum topic for you if one does not exist. 
+
+Keep in mind that you should NOT create forum topics without reasons such as claiming a set, because then you will be the only person (besides admins) that can edit the top post which is usually where important information such as linked hashes or set notes are placed.

```

Comment:
```suggestion
Keep in mind that you should NOT create forum topics without a valid reason, such as claiming a set. Otherwise, you will be the only person (besides admins) who can edit the top post, which is usually where important information such as linked hashes or set notes is placed.
```